medusa/server/api/v2/scene_exception.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+"""Request handler for scene_exceptions."""
+
+from medusa.scene_exceptions import get_last_refresh, retrieve_exceptions
+from tornado.escape import json_decode
+from .base import BaseRequestHandler
+from .... import db
+
+
+def get_last_updates():
+    """Query the cache table for the last update for every scene exception source."""
+    last_updates = {}
+    for scene_exception_source in ['custom_exceptions', 'xem', 'anidb']:
+        last_updates[scene_exception_source] = get_last_refresh(scene_exception_source)[0]['last_refreshed']
+    return last_updates
+
+
+class SceneExceptionOperationHandler(BaseRequestHandler):
+    """Scene Exception request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionOperationHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'PUT')
+
+    def put(self):
```

Comment:
http://dret.typepad.com/dretblog/2011/11/creating-resources-with-get-put.html
"creating resources the RESTful way can be done with either PUT or POST, with PUT expecting that the URI of the created resource is known in advance (by the client issuing the request)"  

http://restcookbook.com/HTTP%20Methods/put-vs-post/
"The HTTP methods POST and PUT aren't the HTTP equivalent of the CRUD's create and update. They both serve a different purpose. It's quite possible, valid and even preferred in some occasions, to use PUT to create resources, or use POST to update resources."

https://jcalcote.wordpress.com/2008/10/16/put-or-post-the-rest-of-the-story/
"Create can be implemented using an HTTP PUT, if (and only if) the payload of the request contains the full content of the exactly specified URL."